src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  list-style: none;
+  margin-left: 16px;
+  padding: 0;
+
+  &__star {
+    background-image: url('/src/images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--#{$stars-quantity} .stars__star:nth-child(-n + #{$stars-quantity}) {
+    background-image: url(/src/images/star-active.svg);
+    background-repeat: no-repeat;
+  }
```

Comment:
Try to use scss loop, because now when i change modifier i don't see result ( you need describe all variants for 1 to 5).